Cargo.toml

```
@@ -12,7 +12,7 @@ readme = "README.md"
 resolver = "2"
 
 [dependencies]
-wgpu = "0.14"
+wgpu = "0.15.1"
```

I'd just update the minor version, and leave the patch out. APIs shouldn't change between patches, so it should be fine.

```suggestion
wgpu = "0.15"
```

---

You're right, resolved.